tests/unit/autosave-proxy-test.js

```diff
@@ -67,3 +67,12 @@ test('changing the content flushes a pending save', function(assert) {
   autosaveObject.set('content', {});
   assert.ok(model.save.called, 'save was called before the content changed');
 });
+
+test('setting a property to the same value', function(assert) {
+  model.set('will-set-same-value', 1);
+  autosaveObject.set('will-set-same-value', 1);
+  assert.ok(!model.save.called, 'save was not called immediately');
+
+  clock.tick(1000);
+  assert.ok(!model.save.called, 'save was called on same value');
```

Comment:
I'm thinking we can remove the earlier assertion and only have this one. Also the assertion message is usually the other way -- saying what should happen. Maybe "save was never called".